### PR TITLE
RBAC - Create User view

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:
The API is not ready yet but we can still merge it.

### CODE REVIEW

File: packages/strapi-admin/admin/src/components/Users/SelectRoles/index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        }}
+        isClearable
+        isDisabled={isDisabled}
+        isMulti
+        options={options}
+        styles={styles}
+        value={value}
+      />
+      {error && value.length === 0 ? (
+        <ErrorMessage style={{ paddingTop: 11, paddingBottom: 0, marginBottom: 17 }}>
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/components/Users/SelectRoles/index.js:54]: We should avoid inline style as much as possible. You can create a component based on ErrorMessage.

File: packages/strapi-admin/admin/src/components/Users/SelectRoles/index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          {translatedError}
+        </ErrorMessage>
+      ) : (
+        <div style={{ height: 11 }} />
```
[the remark this answers is not in the document]
Comment (HichamELBSI) [reply packages/strapi-admin/admin/src/components/Users/SelectRoles/index.js:58]: You can use the padded component here.

File: packages/strapi-admin/admin/src/icons/Duplicate.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Duplicate.defaultProps = {
+  fill: '#0E1622',
+};
+
+Duplicate.propTypes = {
+  fill: PropTypes.string,
+};
+
+export default Duplicate;
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/icons/Duplicate.js:22]: Is it the same icon?
https://github.com/strapi/buffet/blob/develop/packages/buffetjs-icons/src/components/Duplicate/index.js

File: packages/strapi-admin/admin/src/translations/en.json [hunk 4]
```diff
@@ -240,6 +244,14 @@
   "Settings.error": "Error",
   "Settings.permissions.menu.link.roles.label": "Roles",
   "Settings.permissions.menu.link.users.label": "Users",
+  "Settings.permissions.users.add-new": "Add new user",
+  "Settings.permissions.users.create": "Create new User",
+  "Settings.permissions.users.form.email": "Email",
+  "Settings.permissions.users.form.firstname": "First name",
+  "Settings.permissions.users.form.lastname": "Last name",
+  "Settings.permissions.users.listview.header.title": "Users",
+  "Settings.permissions.users.listview.header.description.singular": "0 users found",
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/translations/en.json:253]: This singular message will only be used for the `0 users found`? 
Don't we need a `1 user found`?

File: packages/strapi-helper-plugin/lib/src/components/ModalSection/index.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import themePropTypes from '../../commonPropTypes/themeShape';
+
+const ModalSection = styled.section`
+  display: flex;
+  justify-content: ${({ justifyContent }) => justifyContent};
+  padding: 0 ${({ theme }) => theme.main.sizes.paddings.md};
+`;
```
Comment (HichamELBSI) [inline packages/strapi-helper-plugin/lib/src/components/ModalSection/index.js:9]: It seems that this component is useless as you can use

```javascript
<Flex justifyContent='...'>
     <Padded left right size='md'>
           ....
     </Padded>
</Flex>
```

Am I wrong?

File: packages/strapi-admin/admin/src/components/Users/SelectRoles/IndicatorSeparator.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+const IndicatorSeparator = () => null;
```
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-admin/admin/src/components/Users/SelectRoles/IndicatorSeparator.js:1]: Yes in order to remove the default separator from react select

File: packages/strapi-admin/admin/src/icons/Duplicate.js (hunk 3 above)
Comment (soupette) [reply packages/strapi-admin/admin/src/icons/Duplicate.js:22]: Yes it is I'll update it with the one front buffet!

File: packages/strapi-admin/admin/src/translations/en.json (hunk 4 above)
Comment (soupette) [reply packages/strapi-admin/admin/src/translations/en.json:253]: You are right!

**Pull request merged**